src/Compilers/CSharp/Portable/Binder/Binder.CapturedParametersFinder.cs

```diff
@@ -81,9 +81,7 @@ private CapturedParametersFinder(SynthesizedPrimaryConstructor primaryConstructo
                     // We should define GeneratedNameKind entry (e..g P looks free to use) and add a helper to GeneratedNames that produces the name.
                     // I'd also just keep the name as short as possible to avoid unnecessary metadata bloat. Could be just <name>P.
                     string name = "<" + parameter.Name + ">PC__BackingField";
-
-                    // PROTOTYPE(PrimaryConstructors): Ever read-only?
```

Comment:
>Ever read-only? [](http://example.com/codeflow?start=55&length=15)

Would it be possible to also make the field read-only if it's never mutated? For example in this case:
```cs
public class C(string p) {
    public void M() {
        System.Console.WriteLine(p);
    }
}
```

Reply:
> Would it be possible to also make the field read-only if it's never mutated? 

Theoretically, yes. But we don't mark fields readonly based on how they are used.